### PR TITLE
Sample PR

### PR DESCRIPTION
**Pull request by SonarSourceIT-user1**

### CODE REVIEW

File: src/file-1.xoo [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```
Comment (SonarSourceIT-reviewer1) [inline src/file-1.xoo]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWBam0zcAUnW2zNKg9RI&open=AWBam0zcAUnW2zNKg9RI)

[ISSUE_KEY]: AWBam0zcAUnW2zNKg9RI

File: src/file-2.xoo [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```
Comment (SonarSourceIT-reviewer1) [inline src/file-2.xoo]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWBam0ziAUnW2zNKg9RK&open=AWBam0ziAUnW2zNKg9RK)

[ISSUE_KEY]: AWBam0ziAUnW2zNKg9RK

File: src/file-1.xoo [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// dummy line 11
+// dummy line 12
+// dummy line 13
+// dummy line 14
+// dummy line 15
+// dummy line 16
+// violate HasTag rule: xoo on line 17
```
Comment (SonarSourceIT-reviewer1) [inline src/file-1.xoo]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWBam0zcAUnW2zNKg9RJ&open=AWBam0zcAUnW2zNKg9RJ)

[ISSUE_KEY]: AWBam0zcAUnW2zNKg9RJ

Comment (janos-ss) [reply src/file-1.xoo]: foo

**Review by SonarSourceIT-reviewer1 — APPROVED** [review]

**Pull request closed without merging**